samples/millis/main.cpp

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
This file needs an update to its documentation.

Reply:
Nice catch